web/static/js/components.js

```diff
@@ -2,142 +2,181 @@ import Vue from 'vue'
 
 const spinRate = 16 // milliseconds
 
-let nameForm = Vue.component('name-form', {
-  props: ['name'],
-  template: `
-    <form class="name">
-      <input type="text" v-model="name" maxlength="3" placeholder="___" />
-      <button type="submit" class="send"></button>
-    </form>
-  `
+let spinner = Vue.component('spinner', {
```

Comment:
Why reformat from 2 spaces? That's the convention of the project from the start.